lib/ConstExtract/ConstExtract.cpp

```diff
@@ -148,6 +146,16 @@ static std::shared_ptr<CompileTimeValue> extractCompileTimeValue(Expr *expr) {
       return std::make_shared<ArrayValue>(elementValues);
     }
 
+    case ExprKind::Dictionary: {
+      auto dictionaryExpr = cast<DictionaryExpr>(expr);
+      std::vector<std::shared_ptr<CompileTimeValue>> elementValues;
+      for (unsigned n = dictionaryExpr->getNumElements(), i = 0; i != n; i++) {
```

Comment:
[nit] I think you could use `auto elementExpr : dictionaryExpr->getElements()` here if you didn't want to do the indexing logic.